.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI Build
+on: [push, pull_request]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Download latest NuGet 
+      run: wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
+
+    - name: Install Dependencies
+      run: mono nuget.exe restore
+    
+    - name: Build Project
+      run: xbuild TLSharp.Core/TLSharp.Core.csproj
+      
+  macos:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Download latest NuGet
+      uses: olegtarasov/download-nuget@v1
```

Comment:
same here? maybe curl or wget is already installed in macOS?

Reply:
Done.